src/CMakeLists.txt

```diff
@@ -212,6 +212,13 @@ INSTALL (FILES cnf.sh
                         WORLD_READ
             DESTINATION etc/profile.d)
 
+INSTALL (FILES __fish_command_not_found_handler.fish
```

Comment:
The reason the file is named `__fish_command_not_found_handler.fish` is that it overrides the function with the same name built into fish. (When a function is called in fish, it loads it from the file named <function_name>.fish if such a file exists.)
I could name it `cnf.fish` in the command-not-found source, change this to the following:
```cmake
INSTALL (FILES cnf.fish
            PERMISSIONS OWNER_WRITE
                        OWNER_READ
                        GROUP_READ
                        WORLD_READ
            RENAME __fish_command_not_found_handler.fish
            DESTINATION usr/share/fish/functions)
```
This way, the file would still be installed in the fish directory with the correct name.

Reply:
Does this address your issue correctly?

Reply:
Thanks for clarifying! Yes. That is how I would like to see it then.